packages/vuetify/src/components/VSwitch/VSwitch.tsx

```diff
@@ -79,6 +79,7 @@ export const VSwitch = genericComponent<new <T>(
     const { loaderClasses } = useLoader(props)
     const { isFocused, focus, blur } = useFocus(props)
     const control = ref<VSelectionControl>()
+    const isForcedColorsModeActive = matchMedia('(forced-colors: active)').matches
```

Comment:
This should check the global IN_BROWSER constant and also use window.matchMedia instead of just matchMedia.

Reply:
Done